src/core.c

```diff
@@ -1537,13 +1537,25 @@ void SetWindowSize(int width, int height)
 // Get current screen width
 int GetScreenWidth(void)
 {
-    return CORE.Window.currentFbo.width;
+    return CORE.Window.screen.width;
 }
 
 // Get current screen height
 int GetScreenHeight(void)
 {
-    return CORE.Window.currentFbo.height;
+    return CORE.Window.screen.height;
+}
```

Comment:
I think we expect to get back the logical size from these functions, so it returns the screen size now.
I've created two more functions (GetRenderWidth/GetRenderHeight) in case we want the physical size.
They all return the same size on macos though, maybe I should add this the comment.